OpenRA.Mods.Common/Traits/Wanders.cs

```diff
@@ -118,7 +118,7 @@ void INotifyIdle.TickIdle(Actor self)
 
 		public virtual void DoAction(Actor self, CPos targetCell)
 		{
-			move.ResolveOrder(self, new Order("Move", self, Target.FromCell(self.World, targetCell), false));
+			self.QueueActivity(move.MoveTo(targetCell));
```

Comment:
```suggestion
			self.QueueActivity(move.MoveTo(targetCell, targetLineColor: moveInfo.GetTargetLineColor()));
```

I think we should add a `readonly IMoveInfo moveInfo` variable

Reply:
Actually, we don't need the target line color here since it is not shown.

Reply:
I think it is shown in observer mode. At least in the trait before this change.

Reply:
Added.